src/main/java/duke/controller/TaskManager.java

```diff
@@ -0,0 +1,127 @@
+package duke.controller;
+
+import duke.exception.*;
+import duke.task.*;
+import duke.ui.*;
+
+public class TaskManager {
+    public static void processTask(String input) throws InvalidInputException {
```

Comment:
Try to avoid long methods that goes over 30 lines of code

Reply:
Done! Thank you!